databricks/koalas/frame.py

```diff
@@ -870,17 +870,67 @@ def index(self):
         return Series(self._index_columns[0], anchor=self, index=[])
 
     def set_index(self, keys, drop=True, append=False, inplace=False):
-        """Set the DataFrame index (row labels) using one or more existing columns. By default
-        yields a new object.
-
-        :param keys: column label or list of column labels / arrays
-        :param drop: boolean, default True
-                     Delete columns to be used as the new index
-        :param append: boolean, default False
-                       Whether to append columns to existing index
-        :param inplace: boolean, default False
-                        Modify the DataFrame in place (do not create a new object)
-        :return: :class:`DataFrame`
+        """Set the DataFrame index (row labels) using one or more existing columns.
```

Comment:
not a big deal but this first line looks different from Pandas's - https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L4054